hotsos/core/root_manager.py

```diff
@@ -72,7 +72,18 @@ def data_root(self):
                 if not os.path.exists(target):
                     sys.stdout.write("INFO: extracting sosreport {} to {}\n".
                                      format(path, target))
-                    tar.extractall(self.tmpdir)
+                    try:
+                        tar.extractall(self.tmpdir)
+                    except Exception:
+                        # some members might fail to extract e.g. permission
+                        # denied but we dont want that to cause the whole run
+                        # to fail so we just log a message and ignore them.
+                        tar.errorlevel = 0
+                        sys.stdout.write("INFO: one or more members failed to "
+                                         "extract - disabling error mode and "
+                                         "continuing extraction (which will "
+                                         "be incomplete as a result)\n")
```

Comment:
Don't know if it's worth it to print the exception (except Exception as exc, ...) but this probably doesn't happen often.

Reply:
as things stand currently the logger is not yet setup by this point but its not a bad idea so ill aim to fix that in a separate patch

Reply:
opened https://github.com/canonical/hotsos/issues/739 for ^^